tests/monitoring/vm_monitoring.go

```diff
@@ -81,6 +83,25 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 		})
 	})
 
+	Context("VMI metrics", func() {
+		It("should have kubevirt_vmi_phase_transition_time_seconds buckets correctly configured", func() {
+			vmi := libvmifact.NewGuestless()
+			tests.RunVMIAndExpectLaunch(vmi, 240)
+
+			buckets := virtcontroller.PhaseTransitionTimeBuckets()
+			labels := map[string]string{"le": strconv.FormatFloat(buckets[0], 'f', -1, 64)}
+			libmonitoring.WaitForMetricValueWithLabelsToBe(virtClient, "kubevirt_vmi_phase_transition_time_seconds_bucket", labels, 0, ">", 0)
```

Comment:
In the old location we checked for all buckets existence, why do you only the first now?

Reply:
this was flaky because not always all the histograms buckets were populated, but we know that having 1 VMI, at least the lowest bucket must have a value > 0

Reply:
> not always all the histograms buckets were populated

How so? Buckets are lower-equal, no?

Reply:
+Inf would have the total count? let me try...